downstream/modules/automation-hub/proc-downloading-signature-public-keys.adoc

```diff
@@ -0,0 +1,27 @@
+//this module appears in assembly-collections-and-content-signing-in-pah
+
+[id="proc-downloading-signature-public-keys"]
+
+= Downloading signature public keys
+
+After you sign and approve collections, you must download the signature public keys from the {HubName} UI before you add it to the local system keyring.
+
+.Procedure
+
+. Log in to your private {HubName} instance in the {HubName} UI.
+. In the left navigation, click *Signature Keys*. The Signature Keys dashboard displays a list of multiple keys: collections and container images.
+
+. Select the menu icon, click btn:[Download Key] to download the public key.
++
+* To verify collections, download the key prefixed with `collections-`.
+* To verify container images, download the key prefixed with `container-`.
+
+[NOTE]
```

Comment:
Just a suggestion. These two additional methods of downloading a public key seem too important to have as a note. Is it possible to change the step that begins with "Select the menu icon, click **Download Key**..." to say:

Select one of the following methods to download your public key: 

- Select the menu icon and click btn:[Download Key] to download the public key.
- Select the public key from the list and click the *Copy to clipboard* icon.
- Click the drop-down menu under the *Public key* tab and copy the entire public key block.